packages/auth/src/auth-options.ts

```diff
@@ -37,6 +38,13 @@ declare module "next-auth" {
   }
 }
 
+declare module "next-auth/jwt" {
+  interface JWT extends Record<string, unknown>, DefaultJWT {
```

Comment:
```suggestion
  interface JWT extends Record<string, unknown>, DefaultJWT {
```
This is the nextAuth session-JWT? _Not_ one from auth0?

Reply:
Yes that's correct